### PR TITLE
DOC Small fixes for HQQ and section title

### PR DESCRIPTION
**Pull request by BenjaminBossan**
Changed:

- Helper section had placeholder title
- `device` is not a valid argument to `from_pretrained`
- Excess empty lines

Not sure if py vs python makes a difference in code blocks.

### CODE REVIEW

File: docs/source/package_reference/helpers.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 rendered properly in your Markdown viewer.
 -->
 
-# Document Title
+# Helper methods
```
Comment (stevhliu) [inline docs/source/package_reference/helpers.md:5]: Maybe we can put this section under API reference -> Utilities?

**Review by stevhliu — APPROVED** [review]
LGTM, thanks!

File: docs/source/package_reference/helpers.md (hunk 1 above)
Comment (BenjaminBossan) [reply docs/source/package_reference/helpers.md:5]: Like [this](https://github.com/huggingface/peft/pull/1986/commits/6404cc67957bf5953de7036ce15ddb9ad6ebb10d)?

Comment (stevhliu) [reply docs/source/package_reference/helpers.md:5]: Beautiful! 👍

**Pull request merged**